CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 3.0.0
+
+- Update from stylelint 14.3.0 to 15.10.2
+- Removed deprecated `use-colors` and `use-tokens` rules that were known to cause false positives
```

Comment:
We will no longer need this as releases will be the source of truth :D

Reply:
🎉 Moved into PR description